packages/service-library/src/servicelib/logging_utils.py

```diff
@@ -121,20 +124,41 @@ def config_all_loggers(
     *,
     log_format_local_dev_enabled: bool,
     logger_filter_mapping: dict[LoggerName, list[MessageSubstring]],
+    tracing_settings: TracingSettings | None,
 ) -> None:
     """
     Applies common configuration to ALL registered loggers
     """
-    fmt = DEFAULT_FORMATTING
     the_manager: logging.Manager = logging.Logger.manager
     root_logger = logging.getLogger()
 
     loggers = [root_logger] + [
         logging.getLogger(name) for name in the_manager.loggerDict
     ]
 
+    fmt = DEFAULT_FORMATTING
+    if tracing_settings is not None:
+        fmt = (
+            "log_level=%(levelname)s "
+            "| log_timestamp=%(asctime)s "
+            "| log_source=%(name)s:%(funcName)s(%(lineno)d) "
+            "| log_uid=%(log_uid)s "
+            "| log_oec=%(log_oec)s"
+            "| trace_id=%(otelTraceID)s "
```

Comment:
(but if I am too late with the review, please ignore it :) )

Reply:
No problem. I will add the log prefix
We might need to change this once we get the Grafana setup because its ui might require a different format. But we can do that then

Reply:
Done